src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -29,8 +54,10 @@ export const jiraSymmetricJwtMiddleware = async (req: Request, res: Response, ne
 		const secret = await installation.decrypt("encryptedSharedSecret", req.log);
 
 		const tokenType = checkPathValidity(req.originalUrl) && req.method == "GET" ? TokenType.normal : TokenType.context;
+		let verifiedClaims;
 		try {
-			verifySymmetricJwt(token, secret, req, tokenType, req.log);
+			verifiedClaims = verifySymmetricJwt(token, secret, req.log);
+			verifyJwtClaims(verifiedClaims, tokenType, req);
```

Comment:
I pulled this out `verifyJwtClaims` of `verifySymmetricJwt` function, just because i needed to decode the JWT again and didnt want to rerun that code, this way its available to me for `setJiraAdminPrivileges`